web/modules/custom/dpl_event/dpl_event.module

```diff
@@ -117,3 +118,28 @@ function dpl_event_set_occurred_callback(JobSchedule $job): void {
     $event->save();
   }
 }
+
+/**
+ * Implements hook_form_node_event_form_alter().
+ */
+function dpl_event_form_node_event_form_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
+  // Use the generic "Access" value as the default for the first ticket
+  // category when creating events.
+  $form["field_ticket_categories"]["widget"][0]["subform"]["field_ticket_category_name"]["widget"][0]["value"]["#default_value"] = t('Access', [], ['context' => 'dpl_event']);
```

Comment:
You need to check that this array key already exists, and only set the value if it does.

Also, you match on `[0]` - can there sometimes be more than one item?

Reply:
I updated this with a check that fails hard if the element cannot be located.

This is based on the same reasoning as above.